app/uk/gov/hmrc/customs/declaration/services/BatchFileUploadUpscanNotificationBusinessService.scala

```diff
@@ -63,7 +63,7 @@ class BatchFileUploadUpscanNotificationBusinessService @Inject()(repo: BatchFile
       (bf, ftf) <- maybeFileTransmissionFile(fileReference, md)
     } yield FileTransmission(
       FileTransmissionBatch(md.batchId, md.fileCount),
-      new URL(config.batchFileUploadConfig.fileTransmissionBaseUrl),
+      new URL(config.batchFileUploadConfig.fileTransmissionCallbackUrl + md.csId),
```

Comment:
we usually use interpolation for gluing strings together
```suggestion
      new URL(s"${config.batchFileUploadConfig.fileTransmissionCallbackUrl}${md.csId}"),
```